plza-fe/src/components/locations/LocationEdit.js

```diff
@@ -23,10 +25,12 @@ export default function LocationEdit() {
       );
   };
 
+  // console.log(location);
```

Comment:
Looks like we got a stray console.log to remove 😬

Reply:
Good catch, removed console.log https://github.com/Lambda-School-Labs/lets-get-pizza-fe/pull/71/commits/701b5ac4ec216b0e13ca4988e06c7ba12f922e60